File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupBean.java
package com.streamax.client.ui.devlist.db;

public class GroupBean {
    public String dbAccont;
    public int dbChannel;
    public String dbDevName;
    public int dbFlag;
    public String dbGroupName;
    public int dbId;
    public String dbPwd;

    public String toString() {
        return "DevGroupBean_dbGroupName:" + this.dbGroupName + ",dbDevName:" + this.dbDevName + ",dbChannel:" + this.dbChannel + ",dbId:" + this.dbId + ",dbFlag:" + this.dbFlag;
    }
}

File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupBean.kt
package com.streamax.client.ui.devlist.db

class GroupBean {
    @JvmField var dbAccont: String? = null
    @JvmField var dbChannel: Int = 0
    @JvmField var dbDevName: String? = null
    @JvmField var dbFlag: Int = 0
    @JvmField var dbGroupName: String? = null
    @JvmField var dbId: Int = 0
    @JvmField var dbPwd: String? = null

    override fun toString(): String =
        "DevGroupBean_dbGroupName:$dbGroupName,dbDevName:$dbDevName,dbChannel:$dbChannel,dbId:$dbId,dbFlag:$dbFlag"
}
